Apromore-Frontend/src/bpmneditor/plugins/plugins.js

```diff
@@ -2,6 +2,7 @@ import savePlugin from './apromoreSave';
 import exportPlugin from './export';
 import pdfPlugin from './pdf';
 import sharePlugin from './share';
+import simModelPlugin from './simulateModel';
 import simPanelPlugin from './simulationPanel';
```

Comment:
Can we remove simulationPanel.js and its reference as it's not used any more? (same comment below).